### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by acenasir**
Figma Link Added

### CODE REVIEW

**Comment by MadhushaPrasad** [comment]
cam you please add some valuable text above thr figma link.And also export the Home page and attached it to the readme file

**Comment by MadhushaPrasad** [comment]
@IndudiniThennakoon

File: README.md [hunk 1]
```diff
@@ -87,3 +87,10 @@ You can contribute to the project on github. Here's how you can do it.
 <a href="https://www.w3.org/html/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/html-5.png"/> </a> <a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/css3.png"/> </a><a href="https://www.php.net/" target="_blank"> <img src="https://img.icons8.com/dusk/64/000000/php-logo.png"/>
 
 </br>
+
+ <hr>
+
+<a href="[https://www.w3.org/html/](https://www.figma.com/file/dz4JS3XBiGzTDGGIQ3PczB/Diagon-Alley?node-id=20%3A782)" target="_blank"> Figma Designs </a.
```
Comment (MadhushaPrasad) [inline README.md:93]: The link is not working

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IndudiniThennakoon) [inline README.md:93]: @acenasir link is not working. please update it correctly

**Review by IndudiniThennakoon — CHANGES_REQUESTED** [review]

**Pull request closed without merging**